packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -185,6 +185,7 @@ const metadata: Meta<TableArgs> = {
             <${tableColumnTextTag}
                 column-id="favorite-color-column"
                 field-name="favoriteColor" placeholder="no value"
```

Comment:
It's possible I'm not caught up on our plans, but is there any reason we don't have stories / framework examples / visual tests that set fixed pixel width columns?

Related: do you plan to do the documentation work once this and #1105 go in? Or track that in the backlog?

Reply:
So, my intent was to follow-up your PR with a story focused on column widths. Regarding fixed-width columns, we currently don't have a concrete example of one, and I wasn't intending to create a column for purely Storybook purposes.

Reply:
I added open questions to the future column type work items (icon, numeric, etc) to capture the need for them to decide which width modes to support. There's still an open question about whether the text column will need to support fixed pixel width, but that remains untracked until someone requests it.